### ISSUE TITLE
process exporter does not report status properly

### ISSUE THREAD

**Issue by ndelic0** [ncabatoff/process-exporter#216]
From some reason process exporter does not detect that monitored process is up. 
```
curl -s localhost:9256/metrics | grep -i namedprocess_namegroup_num_procs | grep foo
namedprocess_namegroup_num_procs{groupname="foo"} 0
```
While the monitored process is reported as up in px aux output. 
Only after the process exporter is restarted than it properly catches proper metric value 

```
curl -s localhost:9256/metrics | grep -i namedprocess_namegroup_num_procs | grep foo
namedprocess_namegroup_num_procs{groupname="foo"} 1
```

Any advice?

**Comment by mdschmitt** [comment]
Duplicate of #59 ?

**Comment by hulb** [comment]
> Duplicate of #59 ?

I get the same issue today. I don't think it's duplicated with #59 . The situation is that I'm using process exporter in docker swarm, the process exporter can't get any metrics of process runs on the host which start up later than process exporter. If I recreated the process exporter container then it can see the processes.

**Comment by flixr** [comment]
@hulb see the `-recheck` option: https://github.com/ncabatoff/process-exporter#running

**Comment by hulb** [comment]
> @hulb see the `-recheck` option: https://github.com/ncabatoff/process-exporter#running

Thanks! It works.

**Comment by ndelic0** [comment]
Confirming that using `-recheck` option fixed the issue.
